README.md

```diff
@@ -15,30 +15,7 @@ runners. Both the reconciliation interval and the number of runners to maintain
 
 ## Usage
 
-There are two mandatory configuration options - `path` and `token`.
-
-* `path` determines the organization or repository that the runner will be registered with;
-* `token` is a [GitHub Personal Access Token (PAT)](https://github.com/settings/tokens) (note: this is not the same as the token given in the Add a Runner instructions). The PAT token requires either:
-  * the **`repo`** ("Full control of private repositories") permission for
-use with repositories or;
-  * both the **`repo`** and **`admin:org`** ("Full control of orgs and teams, read and write org projects") permissions for use with an organization. This is necessary because the charm will create and remove runners as needed to ensure that each runner executes only one job to protect jobs from leaking information to other jobs running on the same runner.
-
-The number of runners on a single unit is configured using two configuration options that can be both used at the same time:
-
-* the `containers` option configures the number of LXD container runners;
-* the `virtual-machines` option configures the number of LXD virtual machine runners.
-
-For example, if the charm is deployed with 2 units `juju deploy <charm> -n 2` and the `containers` value of 3 is in use,
-there will be a total of 6 container based runners, three on each unit.
-
-## Reconciliation
-
-Each unit will periodically check the number of runners at the interval specified by `check-interval` to maintain the appropriate number. During the check, all the offline runners are unregistered from GitHub.
-
-If there are more idle runners than configured, the oldest idle runners are unregistered and destroyed. If there are less idle runners than configured, new runners are spawned and registered with GitHub.
-
-During each time period, every unit will make one or more API calls to GitHub. The interval may need to be adjusted if the number of units is large enough to trigger [Rate Limiting](https://docs.github.com/en/rest/overview/resources-in-the-rest-api#rate-limiting).
-
+Please refer to the quick start guide in `docs/tutorial/quick-start.md`.
```

Comment:
Could you refer to it as a tutorial rather than a quick start guide? (A quick start guide implies it will be a how-to guide rather than a tutorial)

Reply:
I have updated to sentence to 
```
Please refer to the tutorial in `docs/tutorial/quick-start.md`.
```

Reply:
Thanks!